src/plugins/data/public/query/query_string/language_service/default_language_reference.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+
+import { EuiButtonIcon, EuiLink, EuiPopover, EuiPopoverTitle, EuiText } from '@elastic/eui';
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { IDataPluginServices } from '../../../types';
+import { useOpenSearchDashboards } from '../../../../../opensearch_dashboards_react/public';
+
+export const DefaultLanguageReference = () => {
+  const opensearchDashboards = useOpenSearchDashboards<IDataPluginServices>();
+  const [isLanguageReferenceOpen, setIsLanguageReferenceOpen] = React.useState(false);
+  const osdDQLDocs = opensearchDashboards.services.docLinks?.links.opensearchDashboards.dql.base;
+  const dqlFullName = (
+    <FormattedMessage
+      id="data.query.queryBar.dqlFullLanguageName"
+      defaultMessage="OpenSearch Dashboards Query Language"
+    />
+  );
+
+  const button = (
+    <div>
+      <EuiButtonIcon
+        iconType={'iInCircle'}
+        aria-label={i18n.translate('discover.queryControls.languageReference', {
```

Comment:
i18n generator will fail because this namespace. Since not in discover. So it should be `data.` something.

Reply:
changed to `data.`